docs/src/rules/no-restricted-imports.md

```diff
@@ -343,7 +395,7 @@ import pick from 'lodash/pick';
 
 Examples of **correct** code for this `group` option:
 
-::: correct { "sourceType": "module" }
+::: correct
```

Comment:
Is there any reason to remove `{ "sourceType": "module" }`? because imports should have `{ "sourceType": "module" }`.

Reply:
This change is reverted and committed.